.travis.yml

```diff
@@ -41,7 +41,7 @@ env:
         # are in astropy-ci-extras. If your package uses either of these,
         # add the channels to CONDA_CHANNELS along with any other channels
         # you want to use.
-        # - CONDA_CHANNELS='astopy-ci-extras astropy'
+        - CONDA_CHANNELS='astropy-ci-extras astropy'
```

Comment:
Sorry, where should I be adding this command to associate it only with the LST builds? Should I add `CONDA_CHANNELS='astopy-ci-extras astropy'` to [this line](https://github.com/astropy/package-template/blob/master/.travis.yml#L74), for example?